code/obj/item/clothing/masks.dm

```diff
@@ -320,6 +323,22 @@
 			return 1
 		return 0
 
+	attack_self(mob/user as mob)
+		if(!src.mask_bald)
+			src.mask_bald = TRUE
+			src.name = "wigless clown mask"
+			src.desc = bald_desc_state
+			src.icon_state = bald_icon_state
+			src.item_state = "clownbald"
+			user.show_text("You tuck back the wig on the [src]")
+		else
+			src.mask_bald = FALSE
+			src.name = initial(src.name)
+			src.desc = initial(src.desc)
+			src.icon_state = initial(src.icon_state)
+			src.item_state = "clown_hat"
+			user.show_text("You untuck the wig from the [src]")
```

Comment:
```suggestion
			user.show_text("You untuck the wig from the [src.]")
```

Reply:
(small correction, put the period after the ] )